module-1/src/Components/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,78 @@
+import { ReactNode } from "react";
+import { createPortal } from "react-dom";
+import "./Dialog.css";
+const mountElement: any = document.getElementById("overlays") || <div />;
+
+interface IDialogProps {
```

Comment:
It's not very common to use `I` prefix for interfaces. I recommend not using that unless it's an agreement in the team/project.
```suggestion
interface DialogProps {
```

Reply:
updated